mackup/applications/musicbrainz-picard.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = MusicBrainz Picard
+
+[configuration_files]
+.config/MusicBrainz
```

Comment:
It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).